src/wasm.cc

```diff
@@ -99,82 +99,46 @@ void WasmBase::registerCallbacks() {
 #undef _REGISTER
```

Comment:
@PiotrSikora should `pthread_equal` also be treated as a capability?

Reply:
No, I'm pretty that sure we don't need it anymore, so feel free to either ignore it or remove it if it gets in your way.